README.md

```diff
@@ -24,6 +24,24 @@ One could also use a standard Clang installation, build a sysroot from the
 sources mentioned above, and compile with
 "--target=wasm32-wasi --sysroot=/path/to/sysroot".
 
+## Install
+
+A typical installation from the release binaries might look like the following:
+```shell script
+wget https://github.com/CraneStation/wasi-sdk/releases/download/wasi-sdk-[VERSION]/wasi-sdk-[VERSION]-linux.tar.gz
+tar xvf wasi-sdk-[VERSION]-linux.tar.gz
+cp -r wasi-sdk-[VERSION] [WASI_SDK_PATH]
```

Comment:
I think I'm ok with this as is, but I wouldn't even include the `cp` command at all.  Mostly because when I use the SDK I just untar it... I don't then copy it.  This seems to imply that copying it is necessary.

I don't mean to be pedantic... do you think it helps to include a copy here?

Reply:
I don't care either way: I originally included it because I have been doing the copy to `/opt/wasi-sdk`... I removed it in the latest push.